public/js/publication-api-entitlements.js

```diff
@@ -90,9 +90,48 @@ async function queryLocalEntitlementsPlans(accessToken, user_id) {
   return entitlementsplans;
 }
 
+/**
```

Comment:
Are these methods still needed / used?

Reply:
Nope, you're right! They've all been moved, so I've deleted them.